rtl/eclrtl/eclrtl.cpp

```diff
@@ -4892,6 +4894,50 @@ class CStrRegExprFindInstance : implements IStrRegExprFindInstance
         }
     }
 
+    bool nextMatch()
+    {
+        if (!matched)
+            return false;
+
+        const char *start = subs[0].second==subs[0].first?subs[0].second+1: subs[0].second;
+        if (start >= sample_end)
+            return (matched = false);
```

Comment:
style: Better as an assignment, then a return - otherwise it could be confused with a comparison.